src/main/kotlin/minesweeper/domain/BoardFields.kt

```diff
@@ -9,7 +9,46 @@ data class BoardFields(val boardFields: List<BoardField>) {
         return BoardFields(nearFields)
     }
 
+    fun aroundNotOpenedNumberFields(coordinate: Coordinate): BoardFields {
+        val nearCoordinate = coordinate.aroundCoordinate()
+        val fields = boardFields.asSequence()
+            .filter { nearCoordinate.contains(it.coordinate) }
+            .filterIsInstance<NumberField>()
+            .filterNot { it.isOpen }
+            .toList()
```

Comment:
```suggestion
    fun aroundNotOpenedNumberFields(coordinate: Coordinate): BoardFields {
        val fields = nearFields(coordinate).notOpenedNumberFields()
```
기존 함수인 nearFields()를 재활용하는 방법도 있을 것 같아요.

Reply:
중간에 제가 변수명을 잘못 설정하였네요! nearCoordinate 대신 aroundCoordinates를 사용해야할 것 같습니다!

nearCoordinate는 주변 8개의 좌표를 의미하지만 aroundCoordinates는 인접한 4개 좌표를 의미하도록 구현하였는데요!
재활용 하기는 쉽지 않을 것 같다는 생각이 드는 것 같습니다 🤔 범준님 생각은 어떠신가요?

제가 명확한 네이밍을 사용하지 못한 것 같기도 하네요 😂

Reply:
아하 제가 로직 파악을 명확하게 못했군요 😅 기존 로직을 재활용하기 쉽지 않을 것 같습니다!

지금은 near - 주위 모든 좌표 / around - 인접한 4개의 좌표 이렇게 구성되어있는 거 같아요.
around의 뜻이 주위라는 걸 생각하면 around - 주위 모든 좌표 / adjacent - 인접한 4개 좌표 이렇게 네이밍을 해보는 건 어떨까 싶네요. 요한님의 생각은 어떠신가요?

Reply:
범준님이 이야기해주신 네이밍을 반영하는 것이 훨씬 명확한 것 같습니다! 👍 👍 
감사합니다 😄